app/screens/about/about.js

```diff
@@ -228,11 +236,34 @@ export default class About extends PureComponent {
                         <FormattedText
                             id='mobile.about.copyright'
                             defaultMessage='Copyright 2015-{currentYear} Mattermost, Inc. All rights reserved'
-                            style={style.footerText}
+                            style={[style.footerText, {marginBottom: 0}]}
                             values={{
                                 currentYear: new Date().getFullYear(),
                             }}
                         />
+                        <FormattedText
+                            id='about.tos_privacy_text'
+                            defaultMessage='{tos} - {privacy}'
```

Comment:
I don't think we need to translate this part since we're already translating the inner links